### ISSUE TITLE
glm::rotation

### ISSUE THREAD

**Issue by sarmazav** [g-truc/glm#570]
Hello.
I have the following test program

```
int main(void){
	glm::mat4 mat = glm::rotate(90.0f,glm::vec3(0.0f,1.0f,0.0f));

	for(int i = 0; i < 4; i++){
		for(int j = 0; j < 4; j++)
			std::cout<<mat[j][i]<<" ";
		std::cout<<std::endl;
	}

	return 0;
}
```

I believe this should print

```
0    0    1    0    
0    1    0    0    
-1   0    0    0    
0    0    0    1  
```

But the output that I get is

```
-0.448074   0      0.893997    0 
0  1  0   0 
-0.893997   0     -0.448074    0 
0   0  0   1
```

I am using g++ (GCC) 6.2.1 if it matters.
And also I am using GLM 0.9.8.1

**Comment by Groovounet** [comment]
GLM is using radians since GLM 0.9.6.
So instead of:
```glm::rotate(90.0f,glm::vec3(0.0f,1.0f,0.0f));```

try:
```glm::rotate(glm::radians(90.0f),glm::vec3(0.0f,1.0f,0.0f));```

**Comment by sarmazav** [comment]
Just to add my 2 cents informing you on something that I just realized, googling for "glm rotate" pops me up with this page https://glm.g-truc.net/0.9.4/api/a00206.html 

That "0.94" in the link is something that only now I saw. Hence the confusion about degrees and radians.
It would be good to make the old documentation pages have a big header or something to clearly indicate to the new users of the library like myself that these docs are old.
